src/main/webapp/dev/js/common/questionRubric.es6

```diff
@@ -10,6 +10,110 @@ import {
     disallowNonNumericEntries,
 } from './ui.es6';
 
+function getRubricChoiceElem(questionNum, col) {
+    return $(`#rubricChoice-${questionNum}-${col}`);
+}
+
+function getRubricWeightElem(questionNum, col) {
+    return $(`#rubricWeight-${questionNum}-${col}`);
+}
+
+function getRubricDescElem(questionNum, row, col) {
+    return $(`#rubricDesc-${questionNum}-${row}-${col}`);
+}
+
+function swapRubricCol(questionNum, colIndex, isSwapLeft) {
+    const $swapCell = $(`#rubric-options-row-${questionNum} .rubricCol-${questionNum}-${colIndex}`);
+    const rubricCellSelector = `td[class*='rubricCol-${questionNum}']`;
+
+    if ((isSwapLeft && $swapCell.prev(rubricCellSelector).length === 0)
+        || (!isSwapLeft && $swapCell.next(rubricCellSelector).length === 0)) {
+        // trying to swap left most or right most column
+        return;
+    }
+
+    const swapValues = ($a, $b) => {
+        const temp = $a.val();
+        $a.val($b.val());
+        $b.val(temp);
+    };
```

Comment:
It may not look good actually. Because, rubric desc needs `row` also. I will use optional parameters to make the code look good. Please review on the new code once I push it.